appserver/tests/payara-samples/samples/rest-management/src/test/java/fish/payara/samples/rest/management/ServerLogTest.java

```diff
@@ -39,24 +39,41 @@
  */
 package fish.payara.samples.rest.management;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 
+import fish.payara.samples.ServerOperations;
+
 import java.util.Scanner;
 
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import org.junit.Before;
 import org.junit.Test;
 
 /**
  * Tests to run against the server log
  */
 public class ServerLogTest extends RestManagementTest {
 
+    @Before
+    public void resetLogging() throws Exception {

```

Comment:
This test consumed logs also from previous executions. Some of them logged warnings.